common/common.scss

```
@@ -525,6 +525,10 @@ img.avatar {
   }
 }
 
+.topic-post.sticky-avatar .topic-avatar {
+    top: calc($--global-nav-height - 0.25em);
```

```suggestion
    top: calc(#{$--global-nav-height - 0.25em);
```

This should work. Since the navigation's height doubles on small screens, I would appreciate it if you could make the necessary adjustment.